### DISCUSSION TITLE
只升级BE到1.1.5数据查不到

### DISCUSSION THREAD

**Discussion by leizi05** [apache/doris#19274] [A - General / Q&A]
将BE升级到1.1.5后，日志显示新版本BE启动成功，且 show proc '/backends'; 显示BE连接成功
但是数据表查不到数据，显示为空，表仍然存在，FE没有升级，将BE退回原版本后，数据可查 原版本显示为 trunk-9800fd3，

**Comment by yagagagaga** [comment]
Upgrading both FE and BE at the same time would be better.
